WordPress/WordPressTest/ReaderDetailViewControllerTests.swift

```diff
@@ -46,22 +46,6 @@ class ReaderDetailViewControllerTests: XCTestCase {
 
 }
 
-/// Builds a ReaderPost
-///
-class ReaderPostBuilder: PostBuilder {
-    private let post: ReaderPost
-
-    override init(_ context: NSManagedObjectContext = PostBuilder.setUpInMemoryManagedObjectContext(), blog: Blog? = nil) {
-        post = NSEntityDescription.insertNewObject(forEntityName: ReaderPost.entityName(), into: context) as! ReaderPost
-    }
-
-    func build() -> ReaderPost {
-        post.blogURL = "https://wordpress.com"
-        post.permaLink = "https://wordpress.com"
-        return post
-    }
-}
-
```

Comment:
This object is used outside of this test file, so it deserves to live in its dedicated file.